### PR TITLE
feat: add footer

### PR DESCRIPTION
**Pull request by PedroMiguelSS**
This PR has the footer. Here is a preview:
screen shot 2018-06-07 at 15 43 34

The font is not the right one as I've created this branch from master before the last PR be merged. 
When both PR's are merged it will be fixed.

### CODE REVIEW

File: src/shared/components/footer/index.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
     <footer className={ classNames(styles.footer, className) }>
-        <div
-            style={ {
-                margin: '0 auto',
-                maxWidth: 960,
-                padding: '1.45rem 1.0875rem',
-            } }>
-            This is the footer
+        <div className={ styles.container }>
+            <div className={ styles.leftContent }>© Protocol Labs | Except as noted, content licensed CC-BY 3.0</div>
```
Comment (fsdiogo) [inline src/shared/components/footer/index.js]: Replace the `©` with `&copy;`.

File: src/shared/components/footer/index.js [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+            <div className={ styles.rightContent }>
+                <div className={ styles.text }>IPFS.JS was started and is sponsored by</div>
```
Comment (vasco-santos) [inline src/shared/components/footer/index.js]: Shouldn't this be translated?

File: src/shared/components/footer/index.js (hunk 1 above)
Comment (vasco-santos) [reply src/shared/components/footer/index.js]: what about translations here?

File: intl/messages/en.json [hunk 3]
```diff
@@ -1,4 +1,6 @@
 {
     "foo": "foo-en",
+    "footerLeftContent": "© Protocol Labs | Except as noted, content licensed CC-BY 3.0",
```
Comment (fsdiogo) [inline intl/messages/en.json]: Replace the `©` with `&copy;`.

File: intl/messages/pt.json [hunk 4]
```diff
@@ -1,4 +1,6 @@
 {
     "foo": "foo-pt",
+    "footerLeftContent": "© Protocol Labs | Traducao Portugues",
```
Comment (fsdiogo) [inline intl/messages/pt.json]: Replace the `©` with `&copy;` and translate to PT.

**Pull request merged**